### PR TITLE
Check return from calloc

### PR DESCRIPTION
**Pull request by johnno1962**
Minor change mopping up PR https://github.com/apple/swift-corelibs-foundation/pull/622 to check a calloc return value by popular demand.

### CODE REVIEW

File: CoreFoundation/Base.subproj/CFUtilities.c [hunk 1]
```diff
@@ -786,9 +786,15 @@ void CFLog1(CFLogLevel lev, CFStringRef message) {
     char *buffer = &stack_buffer[0];
     CFStringEncoding encoding = kCFStringEncodingUTF8;
     CFIndex maxLength = CFStringGetMaximumSizeForEncoding(CFStringGetLength(message), encoding) + 1;
+    const char *tag = "Swift"; // process name not available from NDK
 
     if (maxLength > sizeof(stack_buffer) / sizeof(stack_buffer[0])) {
         buffer = calloc(sizeof(char), maxLength);
+	if (!buffer) {
```
Comment (xwu) [inline CoreFoundation/Base.subproj/CFUtilities.c]: Nit: misaligned indenting

**Comment by parkera** [comment]
@swift-ci please test and merge

**Comment by johnno1962** [comment]
Huh?

**Comment by pushkarnk** [comment]
It's the URLSession test timing out again :(

**Comment by pushkarnk** [comment]
@swift-ci please test

**Comment by johnno1962** [comment]
HI @parkera, can you retry the test please? Apparently this was a timeout.

**Comment by parkera** [comment]
@swift-ci please test and merge

**Comment by johnno1962** [comment]
Thanks

**Pull request merged**